lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -330,6 +357,18 @@ sub _helper_hash {
 	\%helper;
 }
 
+has env => (
+	isa => 'ArrayRef[Str]',
+	is => 'rw',
+	default => sub { [] },
+);
+
+sub _env_hash {
+	my($self) = @_;
+	my %env = map { /^\s*(.*?)\s*=\s*(.*)\s*$/ ? ($1 => $2) : ($_ => '') } @{ $self->env };
```

Comment:
Just wanted to note that on Windows, an empty environment variable won't be passed on to subprocesses (it undefines that variable). See [this](http://stackoverflow.com/questions/1326859/why-dont-the-keys-with-empty-string-values-for-perls-env-show-up-windows-subp). That might trip up people that expect the same behaviour across systems.